pkg/query-service/app/logs/v3/query_builder.go

```diff
@@ -330,17 +307,30 @@ func orderBy(items []v3.OrderBy, tags []string) string {
 	for _, item := range items {
 		if item.ColumnName == constants.SigNozOrderByValue {
 			orderBy = append(orderBy, fmt.Sprintf("value %s", item.Order))
+			addedToGroupBy[item.ColumnName] = true
+		}
+	}
+
+	// add the remaining items
+	if panelType == v3.PanelTypeList {
+		for _, item := range items {
+			// since these are not present in tags we will have to select them correctly
+			if !addedToGroupBy[item.ColumnName] {
+				attr := v3.AttributeKey{Key: item.ColumnName, DataType: item.DataType, Type: item.Type, IsColumn: item.IsColumn}
+				name := getClickhouseColumnName(attr)
+				orderBy = append(orderBy, fmt.Sprintf("%s %s", name, item.Order))
+			}
 		}
 	}
-	return strings.Join(orderBy, ",")
+	return strings.Trim(strings.Join(orderBy, ","), ",")
```

Comment:
nit: this `Trim` is redundant, Join doesn't add a separator at the last https://go.dev/play/p/0UGbGI6Rz7Z